ansible/roles_ocp_workloads/ocp4_workload_serverless/defaults/main.yml

```diff
@@ -3,25 +3,48 @@ become_override: False
 ocp_username: opentlc-mgr
 silent: False
 
-# Defaults values below are for OpenShift Pipelines v1.7.0 (GA)
-
-# Version of kn to be installed on the bastion
-ocp4_workload_serverless_kn_version: "0.13.2"
+# Defaults values below are for OpenShift Serverless v1.7.2 (GA)
 
 # Channel to use for the OpenShift Serverless subscription
 ocp4_workload_serverless_channel: "4.4"
 
-# Set InstallPlan approvel to Automatic? If no it is also suggested to set the starting_csv
-# to pin a specific version
+# Set InstallPlan approvel to Automatic? If no it is also suggested
```

Comment:
approval

Also this comment reads funny. Maybe it is the ? after automatice.

Reply:
OK. Change made. Maybe the update reads better.